graylog2-web-interface/src/components/common/FormikFormGroup.jsx

```diff
@@ -0,0 +1,31 @@
+// @flow strict
+import * as React from 'react';
+
+import FormikInput from './FormikInput';
+
+type Props = {
+  label: string,
+  name: string,
+  type?: string,
+  help?: string,
+  validate?: (string) => ?string,
+};
+
+const FormikFormGroup = ({ label, name, type, help, validate, ...rest }: Props) => (
+  <FormikInput {...rest}
```

Comment:
If you start with `{...rest}` you can never overwrite all the following props, is that intended?

Reply:
In this case I have to structure it like that, otherwise flow complains:
`Cannot spread props because Flow cannot determine a type for  props [1].  rest of object pattern [2] is inexact, so it may contain `value` with a type that conflicts with `value`'s definition in  props [1]. Try making  rest of object pattern [2] exact.`